MapboxCoreNavigation/CLLocation.swift

```diff
@@ -5,13 +5,9 @@ import Turf
 extension CLLocation {
     
     var isQualified: Bool {
-        #if (arch(i386) || arch(x86_64)) && (os(iOS) || os(watchOS) || os(tvOS))
```

Comment:
Now that `isQualified` does not check user course, it's safe to make this check while in the sim.